src/app/issue/settings/settings.component.ts

```diff
@@ -171,8 +263,55 @@ export class SettingsComponent implements OnInit {
   }
 
   addDocumentRow(): void {
-    this.listOfDocuments.push('Relevenates Dokument');
-    console.log(this.listOfDocuments);
+    this.listOfDocuments = [
+      ...this.listOfDocuments,
+      {
+        name: '',
+      },
+    ];
     this.documentRows++;
   }
+
+  //Load all Documents
+  getAllDocuments() {
+    this.issueService.getIssue(this.projectId, this.issueId).subscribe(
+      (data) => {
+        let documents: string[] = data.issueDetail.relevantDocuments;
+        this.listOfDocuments = this.generateRelevantDocumentsArray(documents);
+      },
+      (error) => {
+        console.error(error);
+      }
+    );
+  }
+
+  //Helper method
+  generateStringArray(
+    IssueRelevantDocuments: IssueRelevantDocuments[]
+  ): string[] {
+    let listOfDocuments: string[] = [];
+
+    for (let entry of IssueRelevantDocuments) {
+      listOfDocuments.push(entry.name);
+    }
+
+    return listOfDocuments;
```

Comment:
Könnte man auch in einer Zeile schreiben:
```suggestion
    return IssueRelevantDocuments.map(i => i.name);
```